### DISCUSSION TITLE
Swedish locale - Option to use numbers instead of text for numbers between 1 to 12

### DISCUSSION THREAD

**Discussion by erikdroh** [js-fns/date-fns#1909] [General] [answered]
In Sweden we have a rule regarding numbers, it's that numbers written in text should display as text rather than the number digit.

For example: 
1 => 'en/ett'
2 => 'två'
3 => 'tre' 
etc. up to twelve

This rule has been applied for this library but with no option to only get a numeral translation.

It's very rare to use this formatting for web and apps when we display date or dateFrom formatting. 

Can someone who has worked with this libs locale please fix this or add an option to only use digits instead of text?

Maybe @ejulen knows?

**Comment by ejulen** [comment, marked as answer]
I made the initial localization a rather long time ago, and quite a few updates have been made, so please forgive me if I'm not entirely up to date. What scenario is giving you trouble? It seems to me like the only function related to your question is `formatDistance` (which at least partly already does what you want with the `onlyNumeric` option, albeit only for plural)?

**Reply by erikdroh** [reply]
Ah thanks for the clarification! I don't think onlyNumeric is listed anywhere in the documentation, that's why I couldn't find any information regarding my problem!

Works like a charm :) Thank you

**Comment by ejulen** [comment]
Ah, didn't realize that. Looking at the source code it still looks like you might have an issue for values of 1, though, as the option mentioned only seems to apply to counts of 0 and >1. Might need a PR.
